### PR TITLE
Upgrade to Openssl 1.1.1n

### PR DESCRIPTION
**Pull request by RRathna**
Signed-off-by: Rathna Ramesh <rathna1993@gmail.com>

### CODE REVIEW

**Comment by RRathna** [comment]
bors try

**Comment by RRathna** [comment]
bors try

**Comment by RRathna** [comment]
bors try--

**Comment by RRathna** [comment]
bors try

File: tests/openssl/enc/enc.c [hunk 1]
```diff
@@ -40,11 +41,22 @@ int enc_test(int argc, char** argv, char** env)
...
-
     if (mount("/", "/", OE_HOST_FILE_SYSTEM, 0, NULL))
         goto done;
 #endif
 
+    if (strcmp(argv[0], "ec_internal_test") == 0)
+    {
+        umount("/");
+        if (mount(
+                "/home/ratrames/dev/openenclave/build/tests/openssl/host",
```
Comment (radhikaj) [inline tests/openssl/enc/enc.c:52]: How would this pass in CI?

Comment (RRathna) [reply tests/openssl/enc/enc.c:52]: Thank you for point this out!

**Comment by RRathna** [comment]
bors try

File: tests/openssl/enc/enc.c [hunk 2]
```diff
@@ -40,11 +41,18 @@ int enc_test(int argc, char** argv, char** env)
      */
     if (oe_load_module_host_file_system() != OE_OK)
         goto done;
-
     if (mount("/", "/", OE_HOST_FILE_SYSTEM, 0, NULL))
         goto done;
 #endif
 
+    if (strcmp(argv[0], "ec_internal_test") == 0 ||
```
Comment (mingweishih) [inline tests/openssl/enc/enc.c:48]: Please add comments on why we need to do so.

**Review by mingweishih — APPROVED** [review]

**Comment by RRathna** [comment]
bors try

**Comment by RRathna** [comment]
bors retry

**Comment by RRathna** [comment]
bors try

**Comment by RRathna** [comment]
bors try-

**Comment by radhikaj** [comment]
Bors try

**Comment by radhikaj** [comment]
Bors r+

**Review by radhikaj — APPROVED** [review]

**Pull request merged**